test_runner/src/main/kotlin/ftl/args/IosArgs.kt

```diff
@@ -64,11 +64,8 @@ class IosArgs(
     override val testShardChunks: List<List<String>> by lazy {
         val validTestMethods = Xctestrun.findTestNames(xctestrunFile)
```

Comment:
I think we want to add this:
https://github.com/TestArmada/flank/blob/master/test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt#L77